### ISSUE TITLE
Potential automated request detected - web scraping

### ISSUE THREAD

**Issue by Sylvester-C** [Feodor2/Mypal#384]
Running XP 64-bit at the moment for testing purposes.  Installed Full Moon a week ago and it works fine on 99.9% of sites I have visited.  Then, one site that I visit from time to time blocked me out completely.  This is what is displayed:

Potential automated request detected!  We are making changes to our website therefore web scraping is no longer supported.

http://www.bom.gov.au/

What is web scraping?  Why is this happening with Full Moon?  Do you get the same answer?

**Comment by Feodor2** [comment]
What is the Full Moon? I do not deal with it and do not familiar at all, so can't say something about.

I checked your site with the mypal - works normally seems -no web scraping. For computer terms good to use look on https://en.wikipedia.org/wiki/Web_scraping

**Comment by Sylvester-C** [comment]
I have made a dreadful mistake, sorry.

I have uninstalled the "New Moon" browser and completely deleted every file that it had off my drive and the same in the registry.  It was full of malicious programing, key logging, website redirecting to terrible sites.  I am sure it was tracking and logging everything I did and sending it back to someone.

I now cannot remember where I got it from last week.  The site link was dated a few years ago.  As with your browser, it was a fork of the Pale Moon browser.  I have searched github and cannot find any browser named New Moon.
